hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -794,29 +820,81 @@
       tests: []
     }
     {
-      name: chip_kmac_sram_uncorrectable_alert
-      desc: '''Verify the SRAM uncorrectable alert from KMAC.
+      name: chip_kmac_app_keymgr
+      desc: '''Verify the keymgr interface to KMAC.
 
-            Inject 2 bit errors within the SRAM inside KMAC via backdoor and ensure that this alert
-            propagates to an NMI.
+            // TODO enhance description of keymgr operation
+            - Configure the KMAC to start a KMAC-256 hash with sideloading enabled.
+            - Configure the keymgr to start sending known message data to the KMAC.
+            - Keymgr should transmit a sideloaded key to the KMAC as well.
+            - KMAC should finish hashing successfully (not visible to SW) and return digest to
+              keymgr.
+            - This digest is compared against the known digest value for correctness.
+            - Verify that the keymgr has received valid output from the KMAC.
+
+            X-ref'ed with keymgr test.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_kmac_sram_data_parity_alert
-      desc: '''Verify the data parity alert from KMAC.
+      name: chip_kmac_app_lc
+      desc: '''Verify the LC interface to KMAC.
 
-            Details TBD.
+            // TODO enhance description of LC operation
+            - Configure the KMAC to start a CSHAKE-128 hash.
```

Comment:
I believe this point is not required, since the KMAC interface will know what to compute based on which backdoor interface is sending the request (the same holds for the keymgr and rom_ctrl interfaces).

Reply:
Removed, thanks!